public/javascripts/global.js

```diff
@@ -25,6 +25,11 @@ $(document).ready(function() {
 
     $('body').on('click', '.email-img', appendMessage);
 
+    $('.tabs').hover(
```

Comment:
It would be better to do this in css for the hover state. Also you should add cursor: pointer.